src/System.Management.Automation/engine/runtime/CompiledScriptBlock.cs

```diff
@@ -570,25 +587,23 @@ internal static void CacheScriptBlock(ScriptBlock scriptBlock, string fileName,
                 return;
             }
 
-            //
             // Don't cache scriptblocks that have
             // a) dynamic keywords
             // b) 'using module' or 'using assembly'
             // The definition of the dynamic keyword could change, consequently changing how the source text should be parsed.
             // Exported types definitions from 'using module' could change, we need to do all parse-time checks again.
             // TODO(sevoroby): we can optimize it to ignore 'using' if there are no actual type usage in locally defined types.
-            //
 
             // using is always a top-level statements in scriptBlock, we don't need to search in child blocks.
-            if (scriptBlock.Ast.Find(ast => IsUsingTypes(ast), false) != null ||
-                scriptBlock.Ast.Find(ast => IsDynamicKeyword(ast), true) != null)
+            if (scriptBlock.Ast.Find(ast => IsUsingTypes(ast), false) != null
+                || scriptBlock.Ast.Find(ast => IsDynamicKeyword(ast), true) != null)
             {
                 return;
             }
 
             if (s_cachedScripts.Count > 1024)
             {
-                s_cachedScripts.Clear();
+                ClearScriptBlockCache();
```

Comment:
This reduces readability - in lines 604 and 618 we use `s_cachedScripts`. Please revert.

Reply:
Should I remove the `ClearScriptBlockCache()` method, then?

If I remove this reference it isn't being used anywhere at all, and it's `internal`.

Reply:
It will not a style fix - code cleanup.